CHANGES.rst

```diff
@@ -228,6 +228,10 @@ There's a frood who really knows where his towel is.
 - Fix for generic title for list tiles introduced in 1.0a7 (fixes `#393`_).
   [fredvd]
 
+- Use jQuery's .on instead of deprecated .live method. This allows
```

Comment:
this should be moved to the latest release and rewritten to something like "Make package compatible with jQuery/1.9."